apps/src/templates/projects/ProjectsList.jsx

```diff
@@ -149,59 +150,89 @@ class ProjectsList extends React.Component {
     const thumbnailColumn = {
       property: 'thumbnailUrl',
       header: {
-        props: {style: styles.headerCell},
+        props: {
+          style: {
+          ...tableLayoutStyles.headerCell,
+        }},
       },
       cell: {
         format: thumbnailFormatter,
-        props: {style: styles.thumbnailCell}
+        props: {
+          style: {
+          ...tableLayoutStyles.cell,
+        }}
       }
     };
     const standardColumns = [
       {
         property: 'name',
         header: {
           label: commonMsg.projectName(),
-          props: {style: styles.headerCell},
+          props: {
+            style: {
+            ...tableLayoutStyles.headerCell,
+          }},
           transforms: [sortable],
         },
         cell: {
           format: this.nameFormatter,
-          props: {style: styles.nameCell}
+          props: {
+            style: {
+            ...tableLayoutStyles.cell,
+          }}
         }
       },
       {
         property: 'studentName',
         header: {
           label: commonMsg.studentName(),
-          props: {style: styles.headerCell},
+          props: {
+            style: {
+            ...tableLayoutStyles.headerCell,
+          }},
           transforms: [sortable],
         },
         cell: {
-          props: {style: styles.cell}
+          props: {
+            style: {
+            ...tableLayoutStyles.cell,
+          }}
         }
       },
       {
         property: 'type',
         header: {
           label: commonMsg.projectType(),
-          props: {style: styles.headerCell},
+          props: {
+            style: {
+            ...tableLayoutStyles.headerCell,
+          }},
           transforms: [sortable],
         },
         cell: {
           format: typeFormatter,
-          props: {style: styles.cell}
+          props: {
+            style: {
+            ...tableLayoutStyles.cell,
+          }}
         }
       },
       {
         property: 'updatedAt',
         header: {
           label: commonMsg.lastEdited(),
-          props: {style: styles.headerCell},
+          props: {
+            style: {
+            ...tableLayoutStyles.headerCell,
+          }},
           transforms: [sortable],
         },
         cell: {
           format: dateFormatter,
-          props: {style: styles.cell}
+          props: {
```

Comment:
nit: you're creating an extra object here unnecessarily, you can instead do it like this:

```
props: {style: tableLayoutStyles.cell}
```

Reply:
Thanks!